File: backend-parent/qinglian-pojo/src/main/java/com/yychainsaw/vo/PotentialFriendVO.java
package com.yychainsaw.vo;

import io.swagger.v3.oas.annotations.media.Schema;

import lombok.Data;

import java.util.UUID;

@Data
@Schema(description = "PotentialFriendVO模型")
public class PotentialFriendVO {
    @Schema(description = "userId")
    private UUID userId;
    private String nickname;
    @Schema(description = "avatarUrl")
    private String avatarUrl;
    private String similarContent; // 相似的帖子内容片段
}

File: QingLian-app-android/app/src/main/java/com/yychainsaw/qinglianapp/data/model/vo/PotentialFriendVO.kt
package com.yychainsaw.qinglianapp.data.model.vo

data class PotentialFriendVO(
    val userId: String,
    val nickname: String,
    val avatar: String?,
    val commonInterests: List<String>?,
    val reason: String // e.g. "都在练胸肌"
)